lib/shoryuken/queue.rb

```diff
@@ -52,6 +49,21 @@ def fifo?
 
     private
 
+    def set_name_and_url(name_or_url)
+      if name_or_url.start_with?('https://sqs.')
```

Comment:
@dolan Are you using Amazon SQS or "SQS compatible queue systems"? If the latter, that `start_with?('https://sqs.')` won't work.